From: Takashi Iwai <tiwai@suse.de>
Subject: kABI workaround for the new pci_dev.skip_bus_pm field addition
Patch-mainline: Never, kABI workaround for SLE15 & co
References: bsc#1051510

A standard idiom to move the newly added bitfild to the unused position.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 include/linux/pci.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/pci.h
+++ b/include/linux/pci.h
@@ -324,7 +324,6 @@ struct pci_dev {
 						   D3cold, not set for devices
 						   powered on/off by the
 						   corresponding bridge */
-	unsigned int	skip_bus_pm:1;	/* Internal: Skip bus-level PM */
 	unsigned int	ignore_hotplug:1;	/* Ignore hotplug events */
 	unsigned int	hotplug_user_indicators:1; /* SlotCtl indicators
 						      controlled exclusively by
@@ -332,6 +331,7 @@ struct pci_dev {
 #ifndef __GENKSYMS__
 	unsigned int	clear_retrain_link:1;	/* Need to clear Retrain Link
 						   bit manually */
+	unsigned int	skip_bus_pm:1;	/* Internal: Skip bus-level PM */
 #endif
 	unsigned int	d3_delay;	/* D3->D0 transition time in ms */
 	unsigned int	d3cold_delay;	/* D3cold->D0 transition time in ms */
